app/utils/kcidb.py

```diff
@@ -179,7 +179,19 @@ def push_build(build_id, first, kcidb_options, db_options={}, db=None):
     _submit(kcidb_data, kcidb_options)
 
 
+def _get_misc(group, test):
```

Comment:
`misc` can also be found in builds and revisions, so maybe this should be renamed to `_get_test_misc()` to avoid ambibuity.

Reply:
cahnged to `_get_test_misc()`